cypress/integration/accessibility.spec.js

```diff
@@ -256,4 +256,57 @@ describe('Accessibility Requests', () => {
       });
     });
   });
+
+  describe('request status', () => {
+    beforeEach(() => {
+      cy.localLogin({ name: 'ADMI', role: 'EASI_D_508_USER' });
+      cy.visit('/508/requests/6e224030-09d5-46f7-ad04-4bb851b36eab');
+      cy.get('[data-testid="page-loading"]').should('exist');
+      cy.get('[data-testid="page-loading"]').should('not.exist');
+    });
+
+    it('sets the status to In Remediation', () => {
+      cy.get('a').contains('Change').click();
+      cy.get('#RequestStatus-Remediation')
+        .check({ force: true })
+        .should('be.checked');
+      cy.get('button').contains('Change status').click();
+
+      cy.get('dd').contains('In remediation').should('exist');
+      cy.get('a').contains('Home').click();
+      cy.get('th')
+        .contains('Seeded 508 Request')
```

Comment:
This selector seems pretty verbose. It might be worth adding a `data-testid` to the table row and find the `td` child with  the status and date.